cube/feature_hybrid.h

```diff
@@ -19,7 +19,7 @@
 
 // The FeatureHybrid class implements a Bitmap feature extractor class. It
 // inherits from the FeatureBase class
-// This class describes the a hybrid feature vector composed by combining
```

Comment:
OK, you're right. Just subconsciously corrected this obvious typo when reading the comment.